hawc/apps/assessment/templates/assessment/assessment_downloads.html

```diff
@@ -184,6 +184,90 @@ <h3 class="card-header">Ecological</h3>
     </div>
   </div>
 </div>
+
+
+<p class='form-text text-muted mb-3'>
+  Assessment downloads availalbe for team members or higher.
+</p>
+<div class="row">
+<div class="col-xl-6">
+    <h3>Data extractions</h3>
+    <div class="bg-light border shadow p-3">
+        <p><b>Animal bioassy export</b></p>
+        {% url 'animal:api:assessment-full-export' assessment.pk as the_url %}
+        <span class="d-inline-block" style="width:200px">Published Only:</span>
+        <a class="btn btn-sm btn-primary mx-1" href="{{the_url}}?format=xlsx">xlsx</a>
+        <a class="btn btn-sm btn-primary mx-1" href="{{the_url}}?format=tsv">tsv</a>
+        <a class="btn btn-sm btn-primary mx-1" href="{{the_url}}?format=csv">csv</a>
+        <a class="btn btn-sm btn-primary mx-1" href="{{the_url}}?format=json">json</a>
+        <a class="btn btn-sm btn-primary mx-1" href="{{the_url}}?format=html">html</a>
+        <br/>
+        <span class="d-inline-block" style="width:200px">Published & Unpublished:</span>
+        <a class="btn btn-sm btn-success mx-1" href="{{the_url}}?unpublished=true&format=xlsx">xlsx</a>
+        <a class="btn btn-sm btn-success mx-1" href="{{the_url}}?unpublished=true&format=tsv">tsv</a>
+        <a class="btn btn-sm btn-success mx-1" href="{{the_url}}?unpublished=true&format=csv">csv</a>
+        <a class="btn btn-sm btn-success mx-1" href="{{the_url}}?unpublished=true&format=json">json</a>
+        <a class="btn btn-sm btn-success mx-1" href="{{the_url}}?unpublished=true&format=html">html</a>
+        <p class="text-muted mt-1">Help text</p>
+
+        <p><b>Animal bioassy export</b></p>
+        {% url 'animal:api:assessment-full-export' assessment.pk as the_url %}
```

Comment:
Can you make a component for this little collection of spans and links?

```
{% include 'assessment/components/foo.html' with url=the_url %}
```

Something like that, but with a better name :)